index.md

```diff
@@ -14,11 +14,83 @@ toc_sticky: true
 sidebar:
   - title: "Fetal CMR Resources"
     text: "a list of technical resources for fetal cardiac magnetic resonance imaging"
-last_modified_at: 2021-04-08T00:00:00-04:00
+last_modified_at: 2021-04-23T22:00:00-04:00
 ---
 
-**Notice:** This page is intended as a repository of technical resources put together by the [SCMR Fetal CMR and CMR in Pregnancy Special Interest Group](https://scmr.org/members/group.aspx?id=210568). Content will be added following the first Fetal CMR + CMR in Pregnancy SIG Virtual Session on April 9, 2021.
-{: .notice}
+
+## MOG
+
+MOG – metric optimized gating
+
+<img src="https://avatars.githubusercontent.com/u/6586743" alt="MOG logo" title="MOG" class="align-right" style="height:auto;max-height:25vh;max-width:40vh;"  />
```

Comment:
It would be better to define an image class with these attributes in .css rather than repeat them throughout, but doesn't seem worth the effort in this case.

Reply:
Yeah I don't expect too many images/figures to be added in this way